Resolução do exercicio #34

acidiney opened the pull request.

acidiney committed:
Resolução do exercio #17

acidiney committed:
Resolução do exercio #17

acidiney committed:
Fixed StyleCI Error

acidiney committed:
Fixed StyleCI Error

acidiney committed:
Fixed StyledCI API

acidiney committed:
Fixed StyledCI API

acidiney committed:
Fixed StyledCI API

acidiney committed:
Clear Bugs in ArrayInformationAPI

acidiney committed:
TestFix

acidiney committed:
Clear Bugs in ArrayInformationAPI

acidiney committed:
Resolução do exercicio #16

acidiney committed:
Corrigindo StyleCI no exercicio #16

acidiney committed:
Resolve StyleCI

acidiney committed:
PHPunit fixed -> Using Github Url

acidiney committed:
Enviando Correções do Controller

acidiney committed:
PHPunit fixed -> Using Github Url

acidiney committed:
initial

JoseCage committed:
Merge branch 'desafios' into desafios

JoseCage committed:
Merge branch 'desafios' into desafios

acidiney committed:
Novas alterações

acidiney committed:
merge

acidiney committed:
Clear Conflites

acidiney committed:
Exercicio feito2

acidiney committed:
merge

acidiney committed:
merge

acidiney committed:
styleCI

acidiney committed:
styleCI

acidiney committed:
Add Link

acidiney committed:
Adicionado index a pasta app, para listar os arquivos

JoseCage commented:
Muitos commits juntos rsrs.. Deixa testar aqui o código e avaliar.

acidiney commented:
Tenho que criar uma nova branch na proxima vez né

JoseCage commented on app/ConvertjsonToArray/controller.php:
```diff
@@ -33,7 +33,7 @@
         <td>".$base['app'].'</td>
         <td>'.$base['email'].'</td>
-        <td><a href='.$base['url'].'>Open Site</a></td>
+        <td><a target="_blank" href='.$base['url'].'>Open Site</a></td>
```
`target="_blank"` é meio inseguro de usar :boom:.. Viu a minha sugestão na outra vez @acidiney?

acidiney replied:
Claro que vi, isso é do exercicio antigo nem retifiquei mais kkkk

JoseCage commented on app/UpdateFileJson/index.html:
```diff
@@ -0,0 +1,75 @@
+           <hr />
+           <p class="red"> * PREVIEW</p>
+        </div>
+        <iframe src="http://dojo-php.herokuapp.com/ConvertjsonToArray/" width="100%" height="600" style="border:0px"></iframe>        
```
Há mesmo necessidade deste iframe @acidiney ??

acidiney replied:
Era para retornar já a lista do usuario, só para confirmação... Se achares que não posso remover

JoseCage replied:
No teste que fiz, os o `iframe` não era atualizado. Se calhar fica melhor não chamando o `iframe`, apena informar que o usuário foi adicionado e pedir para pessoa voltar à lista/página onde pega todos membros..

acidiney replied:
já resolvi isso faz tempo kkkkk

acidiney replied:
tenta de novo ... ou use o dojo-php.herokuapp.com

JoseCage replied:
@acidiney vi a mudança.. Conforme comentei no #36, por hora ok. Mas precisaremos mais pra frente listar projetos e não tudo que estiver no diretório.

[Meu comentário explicando](https://github.com/codingdojoangola/dojophp/pull/36#issuecomment-345423325).

JoseCage commented on data/codingdojo.json:
```diff
@@ -1,67 +1 @@
-  ]
-}
\ No newline at end of file
+{"app":"Coding Dojo Angola","url":"http:\/\/codingdojoangola.com","email":"geral@codingdojoangola.com","membros":[{"nome":"Lucius Curado","cargo":"Moderador e Organizador","avatar":""},{"nome":"Jos\u00e9 Cage","cargo":"Moderador e instrutor","avatar":""},{"nome":"Doriel Fonseca","cargo":"Instrutor #dojojavascript","avatar":""},{"nome":"Pedro Massango","cargo":"Desenvolvedor Java","avatar":""},{"n ...
```
Porquê uma palavra obscena na lista de membros? :-1:

JoseCage replied:
No caso o último item dentro da chave/array **membros**..

acidiney replied:
Era só os meus testes kkkkkkkk, preguiça aqui

acidiney replied:
Oh kkkk tavas a falar do cuzão kkkkkkkkkk, kkkkkkkkk eu tava vendo um video br quando tava fazendo ai escreve sem percebr kkkkkkk

JoseCage replied:
Tem que prestar atenção antes do commit meu jovem. Pois o termo sendo ofensivo faz-nos passar vergonha "ao vivo" no evento @acidiney 

:smile_cat:

JoseCage commented on src/UpdateFileJson/UpdateFileJsonAPI.php:
```diff
@@ -0,0 +1,50 @@
+        // In the end, I'm add the new data in my file original
+        if(file_put_contents($this->source, $this->alterFile)):
+            return json_encode(["message" => "File updated with success!"], 201); else:
+            return json_encode(["message" => "File not was updated!"], 400);            
```
O correto neste caso é "**File was not updated!**"

acidiney replied:
Concordo, muita inteligência aqui

JoseCage replied:
Só prestar atenção rsrs..

acidiney committed:
index modify

acidiney committed:
Merge pull request #1 from acidiney/project

Project

acidiney committed:
Adicionei o refresh automatico no iframe

acidiney committed:
link modify

scrutinizer-auto-fixer committed:
Scrutinizer Auto-Fixes

This commit consists of patches automatically generated for this project on https://scrutinizer-ci.com

acidiney committed:
Merge pull request #2 from codingdojoangola/scrutinizer-patch-1

Scrutinizer Auto-Fixes

JoseCage commented:
> Tenho que criar uma nova branch na proxima vez né

Foi o que sugeri tempos atrás lá no CDA.. Um branch pra cada desafio e depois dar merge em `desafios`
